js/contextMenu.js

```diff
@@ -0,0 +1,37 @@
+document.getElementById('tabs').addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+  e.stopPropagation();
+
+  if (e.target.classList.contains('tab-input')){
```

Comment:
Is there a reason the menu needs to be only for the tab input? Unless you're planning to add more actions that are specific to the address bar, I think it would be helpful to have this work in the task overly as well.